src/components/SideNav/_side-nav.scss

```diff
@@ -21,30 +19,26 @@
     fill: $ui-01;
   }
 
-  //TODO: remove after we have updated carbon-components-react above 7.3.1
-  &__footer {
-    display: none;
-  }
-
   &__submenu[aria-haspopup='true'],
   .#{$prefix}--side-nav__link,
   a.#{$prefix}--side-nav__link--current > span.#{$prefix}--side-nav__link-text {
     color: $ui-02;
   }
 
   .#{$prefix}--side-nav__link {
+    align-items: center;
+    display: flex;
+    cursor: pointer;
+    font-size: 0.875rem;
     font-weight: 600;
+    letter-spacing: 0.16px;
+    line-height: 1.125rem;
```

Comment:
`body-long-01` appears to accomplish most of the custom type styles here, could we use it instead?

Reply:
I hesitate only because the purpose of that tag is not really aligned with our sidenav link. If at some point styles are added to accommodate body type issue we will be getting those styles as well.

Reply:
Yeah, we may not want to inherit tweaks to body copy within the scope of the sidenav. Sounds good to leave as is 👍